src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -92,7 +92,7 @@ def _to_vespa_tensor_query(self, marqo_query: MarqoTensorQuery) -> Dict[str, Any
 
         if not marqo_query.approximate:
             query['ranking.softtimeout.enable'] = False
-            query['timeout'] = '300s'
+            query['timeout'] = 300 * 1000  # 5 minutes
```

Comment:
You're doing "1000ms" (string) elsewhere. Do both work?

Reply:
Where is the 1000ms string? I don't use that anymore

Reply:
Only int should work now with timeout.